backend/common/src/main/java/ai/verta/modeldb/common/futures/FutureJdbi.java

```diff
@@ -15,6 +16,10 @@
   public FutureJdbi(Jdbi jdbi, Executor executor) {
     this.executor = executor;
     this.jdbi = jdbi;
+    // Ensure that we do not log any sensitive/private data when exceptions are logged
+    this.jdbi
+        .getConfig(StatementExceptions.class)
+        .setMessageRendering(StatementExceptions.MessageRendering.NONE);
```

Comment:
Can we add an automated test?  Any test that throws a `StatementException` we could grab it and make sure `getMessage()` does not contain `"arguments:"`.  I imagine we could even mock something.

Reply:
I'd ask us to ponder what is the ROI of such test. If it's hard to build and rare to break, it might not be worth to add a test. If it's easy to build and likely to break, it might be. Not every test is worth writing and maintaining (opportunity cost!)